onnx/backend/test/case/node/maxpool.py

```diff
@@ -1,512 +1,707 @@
-# Copyright (c) ONNX Project Contributors
```

Comment:
EOL is wrong.

Reply:
vs code does allow switch between EOLs easily.